### DISCUSSION TITLE
Can't open build settings in the Workflow Editor of the app

### DISCUSSION THREAD

**Discussion by andrii-neroznak-eyewa** [codemagic-ci-cd/codemagic-docs#2814] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [x] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Open Workflow Editor of the app.
2. Open build settings

### Expected results

Displays the build settings.

### Actual results

Showing error
"Oh no!
Looks like something broke down. \a\a Errors such as this one are automatically reported back to us. We'll look into it and make sure it doesn't happen again. \aPlease refresh the page to check if the issue has been resolved."

**Comment by dtrdic** [comment, marked as answer]
Hi @andrii-neroznak-eyewa this issue has been resolved. Please refresh your browser and try again
